scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/Task.java

```diff
@@ -380,6 +400,27 @@ public static Task parseJson(JSONObject jsonTask) {
         result.setTag(tag);
         result.setJobId(jobId);
         result.setJobName(jobName);
+
+        long startAtTime = 0L;
+        JSONObject genericInformationsObject = jsonTask.get("genericInformations").isObject();
+        if (genericInformationsObject != null) {
+            JSONValue genericInformationsValue = genericInformationsObject.get("START_AT");
+            if (genericInformationsValue != null) {
+                String genericInformationsStr = genericInformationsValue.isString().stringValue();
```

Comment:
@lpellegr it make sense. I will make the changes into the scheduler core and propagate the new date. I understand that this date will be set automatically for a cron task, but from the studio it is also possible for the user to set the START_AT, should we let this feature as it is or also modify the studio to let the user directly set the scheduled date ?